### ISSUE TITLE
[Accessibility][Keyboard - Property Pane_Ask a question]: Focus indicator is not clear on all the sides over "Always prompt"check box.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2077]
**User Experience** 
Keyboard users will be affected if focus indication is not clearly visible on all the 4 sides of over "Always prompt"check box. Users will face difficulty if there is no clear focus indicator over an interactive control in the application.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE.
2. Navigate to already created dialog tree item and activate it. Such as- Greetings, Actions, EndDialog etc.
3. Click on "+" and select 'Ask a question'. Then select any of the available options except Oauth login.
4. Navigate to property pane appeared at right side of the screen and navigate to "Always prompt" checkbox using Tab key from keyboard.
5. Observer the issue.

**Actual:** 
Focus indicator is not clear on all the sides over "Always prompt"check box. Keyboard focus is visible on the 3 sides of the checkbox.

**Expected:**
Focus indicator should be clearly visible on all the 4 sides over "Always prompt"check box.

**Note:**
Issue is observed for all the checkboxes in the Property Pane.

**MAS Impacted:** [MAS2.4.7](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c})

**Attachment:**
[MAS2.4.7_Checkbox.zip](https://github.com/microsoft/BotFramework-Composer/files/4249448/MAS2.4.7_Checkbox.zip)
![MAS2 4 7_Checkbox](https://user-images.githubusercontent.com/60913131/75239093-12f07880-57e8-11ea-9fea-12f5bc1c82fa.png)

**Comment by corinagum** [comment]
Note to dev: Based off Fabric, it looks like focus is missing from surrounding both the checkbox and text:
![image](https://user-images.githubusercontent.com/14900841/75414806-9e4c3580-58de-11ea-9bbf-6634c81605e5.png)

Check out: https://developer.microsoft.com/en-us/fabric#/controls/web/checkbox

**Comment by beyackle** [comment]
In the latest version, this is a text field and not a checkbox; this issue doesn't appear to be valid any more.

**Comment by corinagum** [comment]
@ashish315 @awalia13, could we get confirmation that this issue can be closed?

**Comment by ashish315** [comment]
As checked the UI has been updated. In the latest version, this is a text field and not a checkbox; this issue doesn't appear to be valid any more. So closing the bug.
